### ISSUE TITLE
Basics/Camera example cam_hal: cam_dma_config(306): frame buffer malloc failedESP-ROM:esp32s3-20210327

### ISSUE THREAD

**Issue by lboue** [m5stack/M5CoreS3#20]
Hello,

I am facing a DMA malloc issue when running last commit [0b94d75](https://github.com/m5stack/M5CoreS3/commit/0b94d75e3cd29bf8ed3625d0326ffcbe4351b3f9) [Basics/Camera](https://github.com/m5stack/M5CoreS3/blob/main/examples/Basics/Camera/Camera.ino
) example:
cam_hal: cam_dma_config(306): frame buffer malloc failedESP-ROM:esp32s3-20210327

### Setup

- Arduino IDE 2.1.0
- W11

### Full serial log:
```
07:37:27.890 -> E (462) cam_hal: cam_dma_config(306): frame buffer malloc failedESP-ROM:esp32s3-20210327
07:37:27.890 -> Build:Mar 27 2021
07:37:27.890 -> rst:0x15 (USB_UART_CHIP_RESET),boot:0x28 (SPI_FAST_FLASH_BOOT)
07:37:27.890 -> Saved PC:0x420929d6
07:37:27.890 -> SPIWP:0xee
07:37:27.890 -> mode:DIO, clock div:1
07:37:27.890 -> load:0x3fce3808,len:0x44c
07:37:27.890 -> load:0x403c9700,len:0xbec
07:37:27.890 -> load:0x403cc700,len:0x2920
07:37:27.890 -> entry 0x403c98d8
07:37:28.025 -> E (181) esp_core_dump_flash: No core dump partition found!
07:37:28.025 -> E (181) esp_core_dump_flash: No core dump partition found!
07:37:28.476 -> E (460) cam_hal: cam_dma_config(306): frame buffer malloc failed
07:37:28.476 -> E (461) cam_hal: cam_config(390): cam_dma_config failed
07:37:28.476 -> E (461) camera: Camera config failed with error 0xffffffff
```

Do you know where this error can come from?

**Comment by LordLeXaT** [comment]
did you enable PSRAM?
![grafik](https://github.com/m5stack/M5CoreS3/assets/55330538/ff9b8a9b-e548-4a4a-9ec8-b8cc64563b35)
otherwise it will not work, as per defined in code "config.fb_location = CAMERA_FB_IN_PSRAM;" the framebuffer is 
located in the PSRAM

**Comment by lboue** [comment]
> did you enable PSRAM? ![grafik](https://user-images.githubusercontent.com/55330538/242071104-ff9b8a9b-e548-4a4a-9ec8-b8cc64563b35.png) otherwise it will not work, as per defined in code "config.fb_location = CAMERA_FB_IN_PSRAM;" the framebuffer is located in the PSRAM.

I was not enabled by default. In fact, I hadn't paid enough attention.
Perhaps it should be indicated in the code to avoid oversights.

> PS: I told you this already in the last issue you opened.
> 
> PPS: before you ask this, If you compile the code, the cam image looks strange, but there is already a pull request ongoing to fix this #19 by @felmue , If you look into the code changes you can see what you may need to change


Thanks. It works fine after this:
![image](https://github.com/m5stack/M5CoreS3/assets/938089/886caab2-1a65-4b27-a0c4-d76f5e506557)
